css/src/components/hero.scss

```diff
@@ -34,6 +34,52 @@ $hero-image-width-widescreen: calc((100% - 2 * $layout-widescreen-gap) * 0.55) !
 		}
 	}
 
+	.hero-content-right {
+		display: flex;
+		flex-grow: 1;
+		padding-block: $hero-padding-xs;
+
+		@include tablet {
+			justify-content: flex-end;
+			padding-block: $hero-padding-sm;
+			padding-inline-start: $hero-padding-xs;
+		}
+
+		@include desktop {
+			width: $hero-content-fluid-width;
+			min-height: $hero-min-height-default;
+			padding-block: $hero-padding-default;
+			padding-inline-start: $hero-padding-sm;
+			padding-inline-end: $hero-padding-sm;
+		}
+
+		.hero-right-card {
+			align-self: flex-start;
+			background: #ffffffd9;
+			color: $black-static;
+			box-shadow: 0 0 50px 0 #17171726;
+			backdrop-filter: blur(20px);
+
+			.theme-high-contrast & {
+				background: #ffffffd9;
+			}
+
+			@include desktop {
+				background: linear-gradient(116deg, #ffffffd9 13.94%, #ffffff59 111.37%);
+				margin-inline-start: $hero-padding-xs;
+			}
+		}
+	}
+
+	&.hero-right-content {
+		@include tablet {
+			flex-direction: row;
+		}
+		@include desktop {
+			flex-direction: row;
```

Comment:
Can you help me understand the need for duplicating the tablet size's `flex-direction` for desktop? Is this to counter an override provided elsewhere for desktop sizes?

Since Atlas is mobile-first, our breakpoints are accumulative as the browser width grows - so the tablet styles also apply to the desktop and widescreen widths.

Reply:
I think so.  But you may be right that its superfluous now.   Will check as I refactor the names.